GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListViewModel.swift

```diff
@@ -6,4 +6,21 @@
 //  Copyright © 2023 Shalva Avanashvili. All rights reserved.
 //
 
+import SwiftUI
 import Foundation
+
+class DogBreedsListViewModel: ObservableObject {
+    var body: some View {
+        EmptyView()
+        
+    }
+}
+
+struct DogBreedsListViewModel_Previews: PreviewProvider {
```

Comment:
Same applies for this. Viewmodel does not have anything related to view :) Please delete entire `DogBreedsListViewModel_Previews`

Reply:
I have done...